src/urllib3/_async/connection.py

```diff
@@ -40,9 +40,9 @@
 
 
 def is_async_mode():
-    """Tests if we're in the async part of the code or if we're bleached"""
+    """Tests if we're in the async part of the code or if we're in unasync"""
```

Comment:
maybe just "Tests if we're in the async part of the code or not" ?

Reply:
yeah that seem better, what about changing the branch name ?